package.json

```diff
@@ -9,7 +9,7 @@
     "node": "^6.x"
   },
   "scripts": {
-    "build": "yarn && NODE_ENV=production webpack -p --bail --progress",
+    "build": "yarn && yarn test && NODE_ENV=production webpack -p --bail --progress",
```

Comment:
Is this deliberate? Or does it maybe belong in a separate commit?

Reply:
it's deliberate, and has been applied to `asset-admin` and `admin` already